### ISSUE TITLE
Visual indicator missing from <figcaption> link on home page when it receives focus

### ISSUE THREAD

**Issue by biljanap** [wet-boew/GCWeb#150]
This link is keyboard accessible but it does not have visual indicator to indicate that it received focus. This would fail Guideline 2.4.7 Focus Visible: “Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible” (http://www.w3.org/TR/WCAG20/#navigation-mechanisms).
Without visible indicators, keyboard users can’t know if an element on the page received the focus and they can’t interact with that element. Keyboard focus indicator is dotted border that we see around links and form elements when we tab to them.
